modules/farm_fd2/src/entrypoints/transplanting/lib.submit.unit.cy.js

```diff
@@ -0,0 +1,410 @@
+      );
+      expect(transplantingLog.relationships.location[1].id).to.equal(
+        result.transplantingLog.relationships.location[1].id
+      );
+      expect(transplantingLog.relationships.location[2].id).to.equal(
+        result.transplantingLog.relationships.location[2].id
+      );
+
+      expect(transplantingLog.relationships.asset[0].type).to.equal(
+        'asset--plant'
+      );
+      expect(transplantingLog.relationships.asset[0].id).to.equal(
+        result.transplantingLog.relationships.asset[0].id
+      );
+
+      expect(transplantingLog.relationships.category.length).to.equal(1);
+      expect(transplantingLog.relationships.category[0].type).to.equal(
+        'taxonomy_term--log_category'
+      );
+      expect(transplantingLog.relationships.category[0].id).to.equal(
+        result.transplantingLog.relationships.category[0].id
+      );
+
+      expect(transplantingLog.relationships.quantity.length).to.equal(5);
+
+      for (let i = 0; i < 5; i++) {
+        expect(transplantingLog.relationships.quantity[i].type).to.equal(
+          'quantity--standard'
+        );
+        expect(transplantingLog.relationships.quantity[i].id).to.equal(
+          result.transplantingLog.relationships.quantity[i].id
+        );
+      }
+    });
+  });
+
+  it('Check the depth quantity--standard', () => {
+    cy.wrap(farmosUtil.getStandardQuantity(result.depthQuantity.id)).then(
+      (depthQuantity) => {
+        expect(depthQuantity.type).to.equal('quantity--standard');
+        expect(depthQuantity.attributes.measure).to.equal('length');
+        expect(depthQuantity.attributes.value.decimal).to.equal(
+          form.depth.toString()
+        );
+        expect(depthQuantity.attributes.label).to.equal('Depth');
+
+        expect(depthQuantity.attributes.inventory_adjustment).to.be.null;
+
+        expect(depthQuantity.relationships.units.type).to.equal(
+          'taxonomy_term--unit'
+        );
+        expect(depthQuantity.relationships.units.id).to.equal(
+          result.depthQuantity.relationships.units.id
+        );
+
+        expect(depthQuantity.relationships.inventory_asset).to.be.null;
+      }
+    );
+  });
+
+  it('Check the speed quantity--standard', () => {
+    cy.wrap(farmosUtil.getStandardQuantity(result.speedQuantity.id)).then(
+      (speedQuantity) => {
+        expect(speedQuantity.type).to.equal('quantity--standard');
+        expect(speedQuantity.attributes.measure).to.equal('rate');
+        expect(speedQuantity.attributes.value.decimal).to.equal(
+          form.speed.toString()
+        );
+        expect(speedQuantity.attributes.label).to.equal('Speed');
+
+        expect(speedQuantity.attributes.inventory_adjustment).to.be.null;
+
+        expect(speedQuantity.relationships.units.type).to.equal(
+          'taxonomy_term--unit'
+        );
+        expect(speedQuantity.relationships.units.id).to.equal(
+          result.speedQuantity.relationships.units.id
```

Comment:
Use a map for the `units` here.  Get the map using one of the `farmosUtil.js` functions.